machine_test.go

```diff
@@ -420,13 +420,17 @@ func TestStartVMM(t *testing.T) {
 }
 
 func TestLogAndMetrics(t *testing.T) {
+	const logLevel = "DEBUG"
```

Comment:
Should be Debug

Reply:
The input log level became case-insensitive (https://github.com/firecracker-microvm/firecracker/commit/75587b8b6f7bfefddff4baa7402ff0121367b2af) recently and the output is always uppercase (https://github.com/firecracker-microvm/firecracker/blob/db84b948461bddcc89c0b69aa5639368a7d7c3ef/src/logger/src/logger.rs#L82).

Reply:
Ah, cool!